src/types/intentions.ts

```diff
@@ -1,8 +1,19 @@
-const AVIARY_INTENTIONS = {
+const CORE_AVIARY_INTENTIONS = {
   primary: "primary",
   system: "system",
   danger: "danger",
   success: "success",
+};
+
+const AVIARY_INTENTIONS = {
+  ...CORE_AVIARY_INTENTIONS,
+  warning: "warning",
```

Comment:
We have a ticket to add `highlight` back in as `autoship`. Should we add this in here too?

Reply:
Good call, added this in!